okio/src/main/java/okio/Utf8.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2017 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okio;
+
+/** APIs for working with UTF-8 strings. */
+public final class Utf8 {
+  private Utf8() {
+  }
+
+  /**
+   * Returns the number of bytes used to encode {@code string} as UTF-8 when using {@link
+   * ByteString#encodeUtf8} or {@link Buffer#writeUtf8(String)}.
```

Comment:
Sounds like the `ByteString#encodeUtf8()` part should be dropped from the JavaDoc. Alternatively, `ByteString#encodeUtf8()` could be changed to manually UTF-8-encode the input string.